### ISSUE TITLE
ERROR:  Error installing bundler on ruby 2.5.3

### ISSUE THREAD

**Issue by meets7** [bundler/bundler#6866]
Bundler installation fails with the following message.
```
ERROR:  Error installing bundler:
The last version of bundler (>= 0) to support your Ruby & RubyGems was 1.17.3. Try installing it with `gem install bundler -v 1.17.3`
bundler requires RubyGems version >= 3.0.0. The current RubyGems version is 2.7.6. Try 'gem update --system' to update RubyGems itself.
```
We are using rbenv to use ruby version 2.5.3. This worked fine till 31st December 2018.

**Comment by skarger** [comment]
@meets7 when you install Bundler you are getting the newly released version 2.0.

See https://bundler.io/blog/2019/01/03/announcing-bundler-2.html

Ruby 2.5.3 is fine. As the error message says, your problem is RubyGems 2.7.6. I advise following the upgrade steps. Or if you cannot upgrade rubygems, install a specific older version of Bundler e.g. `1.17.3`.

**Comment by thbar** [comment]
FWIW, this also affects the RVM ansible role (https://github.com/rvm/rvm1-ansible/issues/193), which unfortunately does not natively support upgrading rubygems right before calling `bundle install`, nor providing a bundler version (all users of the Ansible role will see a failure).

**Comment by indirect** [comment]
There's a fix for this in #6867.
